content/docs/addons-test-utils.md

```diff
@@ -236,7 +236,7 @@ findRenderedDOMComponentWithClass(
 )
 ```
 
-Like [`scryRenderedDOMComponentsWithClass()`](#scryrendereddomcomponentswithclass) but expects there to be one result, and returns that one result, or throws exception if there is any other number of matches besides one.
+जैसे [`scryRenderedDOMComponentsWithClass()`](#scryrendereddomcomponentswithclass) लेकिन एक परिणाम होने की उम्मीद करता है, और एक परिणाम देता है, या एक के अलावा किसी अन्य संख्या के मैच होने पर एक्सेप्शन थ्रो करता है।
```

Comment:
could be reworded to : [`scryRenderedDOMComponentsWithClass()`](#scryrendereddomcomponentswithclass)  ke samaan, lekin...

Reply:
fixed in f6949489e064deef0435f5b70bccff1711585cf2